api/src/main/java/org/openmrs/api/ConditionService.java

```diff
@@ -62,6 +62,16 @@
 	@Authorized({ PrivilegeConstants.GET_CONDITIONS })
 	List<Condition> getActiveConditions(Patient patient) throws APIException;
 
+    /**
+	 * Gets a patient's active and inactive conditions
+	 *
+	 * @param patient - the patient to retrieve conditions for
+	 * @throws APIException   
+	 * @return a list of the patient's conditions
+	 */
+	@Authorized({ PrivilegeConstants.GET_CONDITIONS })
+	List<Condition> getAllConditions(Patient patient) throws APIException;
```

Comment:
Can you also include the since annotation? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NewPublicMethodsandClasses

Reply:
Just like the rest of our API, i thought you would parameterise this to let one decide if they want to include inactive conditions. Can you take a second look at the rest of our API?

Reply:
Did you see the above comment?

Reply:
@dkayiwa  I checked the above comment, but not really sure what you are trying to point out. Could you please make it more clear :)